### PR TITLE
Admin styling

### PR DESCRIPTION
**Pull request by kmthorsnes**
#### Description
Changes proposed in this pull request:

* Adds bootstrap 
* Adds styling to admin pages
#### What I have learned working on this feature: 
* To setup bootstrap for rails applications. 

#### Screenshots:
skjermbilde 2018-04-13 kl 17 35 48
skjermbilde 2018-04-13 kl 17 35 38
skjermbilde 2018-04-13 kl 17 35 29

### CODE REVIEW

**Review by davidboll — APPROVED** [review]
Looks good!

**Review by jyuw — APPROVED** [review]

File: app/views/athletes/index.html.erb [hunk 1]
```diff
@@ -1,67 +1,79 @@
...
+  <%= link_to 'New Athlete', new_athlete_path, class:"btn btn-info" %>
 
-        <%= form_with url: athlete_path(athlete), method: :delete, local: true do |form| %>
-          <%= form.submit value: 'Delete' %>
-        <% end %>
-      </li>
-    <% end %>
-  </ul>
-<% end %>
+  <p></p>
```
Comment (holgertidemand) [inline app/views/athletes/index.html.erb]: WTF

Comment (holgertidemand) [reply app/views/athletes/index.html.erb]: <br>

Comment (holgertidemand) [reply app/views/athletes/index.html.erb]: ``` <br> ```

File: app/views/athletes/index.html.erb [hunk 2]
```diff
@@ -1,67 +1,79 @@
...
-  <%= form.submit value: 'Revert Result' %>
+      <%= form_with url: toggle_voting_path(athlete), method: :put, local: true do |form| %>
+      <%= form.submit value: 'Open/Close Voting', id: 'voting' %>
+    <% end %>
+
+    <p>Status: <strong><%= athlete.votingOpen %></strong></p>
+      <%= link_to 'Edit Athlete', edit_athlete_path(athlete), class: "btn btn-success" %>
+      <%= form_with url: athlete_path(athlete), method: :delete, local: true do |form| %>
+      <button type="submit" class="btn btn-danger">Delete</button>
+    <p></p>
```
Comment (holgertidemand) [inline app/views/athletes/index.html.erb]: WTF

File: app/views/athletes/index.html.erb [hunk 3]
```diff
@@ -1,67 +1,79 @@
...
 <% end %>
+</div>
 
-<h3>Results</h3>
+<div class="results">
+  <h3 id="results">Results</h3>
+    <%= form_with url: publish_path, method: :put, local: true do |form| %>
+    <%= form.submit value: 'Publish Result' %>
+  <% end %>
+  <p></p>
```
Comment (holgertidemand) [inline app/views/athletes/index.html.erb]: WTF

**Review by holgertidemand — CHANGES_REQUESTED** [review]
Please see comments

**Pull request merged**